mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/gateway/tasks/GatewayMigrationTask.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2020, Mulesoft, LLC. All rights reserved.
+ * Use of this source code is governed by a BSD 3-Clause License
+ * license that can be found in the LICENSE.txt file.
+ */
+package com.mulesoft.tools.migration.library.gateway.tasks;
+
+import static com.mulesoft.tools.migration.project.ProjectType.MULE_FOUR_APPLICATION;
+import static com.mulesoft.tools.migration.util.MuleVersion.MULE_3_VERSION;
+import static com.mulesoft.tools.migration.util.MuleVersion.MULE_4_VERSION;
+
+import com.mulesoft.tools.migration.library.gateway.steps.ApiMigrationStep;
+import com.mulesoft.tools.migration.library.gateway.steps.GatewayNamespaceMigrateStep;
+import com.mulesoft.tools.migration.project.ProjectType;
+import com.mulesoft.tools.migration.step.MigrationStep;
+import com.mulesoft.tools.migration.task.AbstractMigrationTask;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Gateway migration task
+ *
+ * @author Mulesoft Inc.
+ */
+public class GatewayMigrationTask extends AbstractMigrationTask {
```

Comment:
@Pgarrett this task is not being executed on the latest version of MMA. Should we leave as it is?

Reply:
Hi @julianpascualMuleSoft! This task can be removed, it is not being used